wondrous-app/components/Common/KanbanBoard/kanbanBoard.tsx

```diff
@@ -34,7 +34,7 @@ const populateOrder = (index, tasks, field) => {
     aboveOrder = tasks[index - 1][field];
   }
   if (index < tasks.length - 1) {
-    belowOrder = tasks[index + 1][field];
```

Comment:
@andywong418  im not sure if this is right or not because I'm not too familiar with the logic here. But from testing this is right. can you check the logic for me.

Reply:
If this looks right from testing then this should be fine, perhaps because we zero index things? As the index should be the index of the current task...Not sure why above order would be index-1 then

Reply:
lol idk either